app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -415,6 +416,20 @@ class SamplesHeatmapView extends React.Component {
       this.fetchMetadataFieldsBySampleIds(),
     ]);
 
+    const pipelineMajorVersionsSet = new Set(
+      compact(
+        map(data => {
+          if (!data.pipeline_version) return null;
```

Comment:
When are pipeline versions null? I noticed some older samples that displayed v2.11 on the DD table were missing the pipeline version here, so the warning didn't display on the heatmap.